### ISSUE TITLE
Infobar ribbon "This is a demo journal..."

### ISSUE THREAD

**Issue by Maelplaine** [libero/publisher#169]
## Problem / Motivation
 
- WHO eLife's tech, product and marketing teams
- WHEN and WHERE at the top of the page
- WHAT a banner or ribbon informing the visitors that this website is a demo journal
- WHY we do not want visitors to confuse this website for a real journal and we want as well them to understand that this is only a demo instance of a product that is still in progress

## Proposed solution

- We could reuse the info-bar pattern to display the message
- This ticket is linked to: https://github.com/libero/libero/issues/171
- The banner message has to be short so that it looks fine on mobile devices
- Proposed message: "This is a Libero Publisher demo journal"
The banner could be orange.

## Tasks

- [ ] A descriptive of a task
- [ ] A description

## Clarification needed and assumptions

- Is the banner message signed off?
> Approved by Jen
- Should the message explain that this is the first iteration of Libero Publisher?
> Let's keep the above copy for now and see how people react
- Which colour should have the banner? Is orange ok?
> Orange is fine

## Technical notes
- We could use this technology 
- More notes or suggestions

**Comment by davidcmoulton** [comment]
Blocked by #171 (info bar pattern).

**Comment by thewilkybarkid** [comment]
Agreed not to include a link.

**Comment by thewilkybarkid** [comment]
https://github.com/libero/sample-configuration/pull/76

**Comment by Maelplaine** [comment]
@thewilkybarkid I've created a new ticket for the missing link #216
